2324-wt/abstract-interpretation/interim/poster.tex

```diff
@@ -0,0 +1,221 @@
+\makeatletter
+\newcommand\input@path{{smile},{flowr-logo}}
+\makeatother
+
+\documentclass[color,coloraccent=red!60!black,listings]{poster}
+
+\usepackage[notext,color=accent]{flowrlogo}
+\usepackage{lipsum}
+\usepackage[normalem]{ulem}
+\usepackage[verbatim]{lstfiracode}
+\usepackage[notext,nott,nosf,nomath]{kpfonts-otf}
+
+\title{Abstract Interpretation}
+\subtitle{The cooler AI}
+\author{Lukas Pietzschmann, Florian Sihler}
+\uni{Ulm University}
+\institute{Institute of Software Engineering and Programming Languages}
+\date{02/02/2024}
+\logo{logos/splogo.png}
```

Comment:
soll das so bleiben? wenn ja, dann könnte man vlt noch dafür sorgen, dass sich die Höhe/Position an irgendetwas orientiert oder es vlt auch aussieht wie ne bookmark?

Reply:
Ich bin mir nicht ganz sicher, was du mit der ersten Hälfte des zweiten Satzes meinst. Der Kasten, in dem das Logo ist, ist ja nicht _random_ platziert. Und das Logo im Kasten ist einfach mittig.

Aber wir können gerne ausprobieren, wie ein Lesezeichen aussieht :)

Reply:
75101fa hat das Lesezeichen hinzugefügt